compose/services.yml

```diff
@@ -162,3 +165,17 @@ services:
       BALENA_HAPROXY_KEY: ${OPENBALENA_ROOT_KEY}
       BALENA_ROOT_CA: ${OPENBALENA_ROOT_CA}
       HAPROXY_HOSTNAME: ${OPENBALENA_HOST_NAME}
+    volumes_from:
+      - cert-provider
+
+  cert-provider:
+    build: ../cert-provider
+    volumes:
+      - certs:/certs
+      - cert-provider:/usr/src/app/certs
+    environment:
+      DOMAINS: "api.${OPENBALENA_HOST_NAME},registry.${OPENBALENA_HOST_NAME},s3.${OPENBALENA_HOST_NAME},vpn.${OPENBALENA_HOST_NAME}"
+      EMAIL: ${OPENBALENA_SUPERUSER_EMAIL}
+      OUTPUT_PEM: /certs/open-balena.pem
+      # ACTIVE: "true"
+      # PRODUCTION: "true"
```

Comment:
Should these two be passed from the environment like the others (eg. `OPENBALENA_HOST_NAME`)?

Reply:
I have left them commented as it's an optional service.

Reply:
@dfunckt done